cognite/v05/timeseries.py

```diff
@@ -728,6 +728,36 @@ def _get_datapoints_frame_user_defined_limit(time_series, aggregates, granularit
     return df
 
 
+def post_datapoints_frame(data, **kwargs):
+    """Write a dataframe 
+
+    Args:
+        dataframe (DataFrame):  Pandas DataFrame Object containing the timeseries
+
+    Keyword Args:
+        api_key (str): Your api-key.
+
+        project (str): Project name.
+
+    Returns:
+        An empty response.
+    """
+    api_key, project = config.get_config_variables(kwargs.get("api_key"), kwargs.get("project"))
+
+    try:
+        timestamp = data.timestamp
+        names = data.drop(['timestamp'], axis=1).columns
+    except:
+        print('DataFrame not on correct form')
```

Comment:
Instead of printing to stdout and returning -1, you should raise cognite._utils.InputError with your error message as argument.